packages/web-react/.eslintrc.js

```diff
@@ -65,5 +65,7 @@ module.exports = {
     'import/no-extraneous-dependencies': ['error', { packageDir: ['./', '../../'] }],
     // disable double quotes
     quotes: ['warn', 'single'],
+    // allow only forwarded reference for specific components
+    'no-underscore-dangle': ['error', { allow: ['_Button', '_ButtonLink'] }],
```

Comment:
Why here? I would prefer to have this comment directly where is the problem and with an explanation of why we allow it.

Reply:
We ended up not using this so I'm deleting it

Reply:
Because we need to make an exception for components that are composed before they are exported with forwardRef. That's why @literat and I made this exception. Where and why would you still like to put it?
I think this is enough.